--- a/glxdemos/CMakeLists.txt
+++ b/glxdemos/CMakeLists.txt
@@ -19,7 +19,6 @@ target_link_libraries(glxinfo ${OPENGL_gl_LIBRARY} ${X11_X11_LIB}
 if(CMAKE_SYSTEM_NAME STREQUAL "SunOS")
 	target_link_libraries(glxinfo Xmu)
 endif()
-install(TARGETS glxinfo DESTINATION ${CMAKE_INSTALL_BINDIR})
 
 if(VGL_EGLBACKEND)
 	add_executable(eglinfo eglinfo.c glinfo_common.c)
@@ -28,9 +27,6 @@ if(VGL_EGLBACKEND)
 endif()
 
 set(GLXSPHERES glxspheres)
-if(BITS EQUAL 64)
-	set(GLXSPHERES glxspheres64)
-endif()
 add_executable(${GLXSPHERES} glxspheres.c)
 target_link_libraries(${GLXSPHERES} ${OPENGL_gl_LIBRARY} ${X11_X11_LIB}
 	${OPENGL_glu_LIBRARY} m)
